client/delegate-auth/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "delegate-auth",
+  "version": "1.0.0",
+  "description": "Sample example of a nodejs application which authenticates to the example server examples with a key which is not the owner of the DID but is a authentication delegate of the DID.",
+  "main": "delegate-auth.js",
+  "scripts": {
+    "script1": "ts-node delegate-auth-script.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@ew-did-registry/jwt": "^0.5.2-alpha.90.0",
+    "@ew-did-registry/keys": "^0.5.2-alpha.90.0",
+    "axios": "^0.21.4",
+    "ethers": "^5.4.6",
+    "iam-client-lib": "3.0.0",
+    "node-emoji": "^1.11.0",
+    "ts-node": "^10.2.1",
+    "typescript": "^4.4.2"
+  },
+  "devDependencies": {
+    "@types/node-emoji": "^1.8.1",
+    "jest": "^27.2.1"
```

Comment:
I don't actually see any jest tests. Is this dependency needed?

Reply:
No I think I added it before running tests on passport-did-auth

Reply:
```suggestion

```
It would still be good to remove the dependancy then